.circleci/config.yml

```diff
@@ -1,5 +1,13 @@
 version: 2
 jobs:
+  litmus_nightly:
+    machine: true
+    steps:
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run: docker run --entrypoint "./run_litmus_tests_oss.sh" -e TEST_LIST=tests_lists/gateway_api_tests.list --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/Litmus/result quay.io/influxdb/litmus:latest
+      - store_artifacts:
+          path: ~/project
```

Comment:
Do you have test results in this directory, if so those could be exposed differently in the ui by using `store_test_results`: https://circleci.com/docs/2.0/configuration-reference/#store_test_results

Reply:
I will look into it. Thanks